Resources/Prototypes/Entities/Clothing/Shoes/base_clothingshoes.yml

```
@@ -51,3 +51,16 @@
     interfaces:
     - key: enum.StorageUiKey.Key
       type: StorageBoundUserInterface
+
+- type: entity
+  abstract: true
+  parent: ClothingShoesBase
```

Winter boots were butcherable before, so it should stay that way. 
```suggestion
  parent: ClothingShoesBaseButcherable
```

---

Okay.

---

I guess I don't need this component 
- type: Butcherable
    butcheringType: Knife
    spawned:
    - id: MaterialCloth1
      amount: 1 
in ClothingShoesBaseWinterBoots because it is already in the parent ClothingShoesBaseButcherable?
Did I understand everything correctly?